src/Raku/Actions.nqp

```diff
@@ -2150,6 +2150,15 @@ class Raku::Actions is HLL::Actions does Raku::CommonActions {
     method pod_block:sym<finish>($/) {
         $*CU.replace-finish-content($*LITERALS.intern-str(~$<finish>));
     }
+
+    method comment:sym<line_directive>($/) {
+        my $origin-source := $*ORIGIN-SOURCE;
+        my $orig-line := $origin-source.original-line($/.from());
+        $origin-source.register-line-directive(
+            $origin-source.original-line($/.from()),
```

Comment:
Shouldn't this just be the `$orig-line` declared above?

Reply:
Actually, it's the `$orig-line` which is unnecessary. Fossils of a debug print.